app/gui2/src/stores/graph/graphDatabase.ts

```diff
@@ -160,9 +160,6 @@ export class GraphDb {
 
   connections = new ReactiveIndex(this.bindings.bindings, (alias, info) => {
     const srcNode = this.getPatternExpressionNodeId(alias)
-    // Display connection starting from existing node.
-    //TODO[ao]: When implementing input nodes, they should be taken into account here.
-    if (srcNode == null) return []
```

Comment:
I'm not sure about this change. What about nodes with lambdas? We should omit the "connection" from lambda parameter to its usage.

Reply:
True, fixed it.